calyx-py/calyx/builder.py

```diff
@@ -446,6 +446,24 @@ def comb_mem_d2(
             is_external,
             is_ref,
         )
+    
+    def comb_mem_dn(
```

Comment:
Consider a more descriptive name. `dn` reads like "down" to me! Also flesh out the docstring to explain what's up.